pkg/ext-proc/metrics/metrics_test.go

```diff
@@ -88,6 +90,65 @@ func TestRecordRequestCounterandSizes(t *testing.T) {
 	}
 }
 
+func TestRecordRequestErrorCounter(t *testing.T) {
+	type requests struct {
+		modelName       string
+		targetModelName string
+		error           string
+	}
+	scenarios := []struct {
+		name    string
+		reqs    []requests
+		invalid bool
+	}{{
+		name: "multiple requests",
+		reqs: []requests{
+			{
+				modelName:       "m10",
+				targetModelName: "t10",
+				error:           errutil.Internal,
```

Comment:
We don't record the error for invalid request or if I miss something? Was it because we don't have model or target model name?

Reply:
This unit test file just tests simple functionality that it will accumulate error counts if the stream is the same. For invalid request error I will cover it in the integration test.